modules/rhcos-install-iscsi-ibft.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-install-iscsi-ibft_{context}"]
+= Installing {op-system} on an iSCSI boot device using iBFT
+
+On a completely diskless machine, the iSCSI target and initiator values can be passed through iBFT. iSCSI multipathing is also supported.
+
+.Prerequisites
+. You are in the {op-system} live environment.
+. You have an iSCSI target you want to install {op-system} on.
+. Optional: you have multipathed your iSCSI target.
+
+.Procedure
+
+. Mount the iSCSI target from the live environment:
```

Comment:
```suggestion
. Mount the iSCSI target from the live environment by running the following command:
```
I see that this whole doc section does not use the current style, but I think these new modules should.  I would add "by running the following command" to every step where it is relevant.

Reply:
good point!